starter-code/index.js

```diff
@@ -1,17 +1,74 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items=[];
+    this.length=0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function(a, b) {
+      return a - b;
+    });
+    this.length = this.items.length;
+  }
+   
+  get(pos) {
+    let result;
+    if (pos > this.length-1){
+      throw new Error("OutOfBounds");
+    } else {
+      this.items.forEach(function (el, i){
+        if (pos === i) {
+          result = el;
+        } 
+      });
+    }
+    return result;
+  }
 
-  get(pos) {}
+  max() {
+    let max;
+    if (this.length === 0){
+      throw new Error("EmptySortedList");
+    } else {
+      max = this.items.reduce(function(a, b) {
+        return Math.max(a, b);
```

Comment:
no hace falta que hagas un reduce, para retornar un Math.max, si le pasas como argumento, los valores de un array, ya te vale `Math.max(...this.items)`